src/client/systems/views/starsystemview.cpp

```diff
@@ -780,7 +785,9 @@ void SysStarSystemRenderer::CalculateCityPositions() {
             continue;
         }
         auto& coord = m_app.GetUniverse().get<cqspt::SurfaceCoordinate>(city_entity);
-        m_app.GetUniverse().emplace_or_replace<Offset>(city_entity, cqspt::toVec3(coord,  1));
+        cqspb::Body parent = m_app.GetUniverse().get<cqspb::Body>(m_viewing_entity);
+        m_app.GetUniverse().emplace_or_replace<Offset>(
+            city_entity, cqspt::toVec3(coord, parent.radius/50000));
```

Comment:
Because the offset is supposed to be an offset coordinate and if we do not provide the function with the correct unit the results will be off.
This will actually eventually need to be moved further out of the render code to the simulation tick.